cmd/otelcontribcol/builder-config.yaml

```diff
@@ -50,6 +50,7 @@ extensions:
     import: github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/textencodingextension
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/zipkinencodingextension v0.104.0
     import: github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/zipkinencodingextension
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/cfgardenobserver v0.104.0
```

Comment:
Nit: Could this line be moved up to be with the other observer extensions, placing it in alphabetical order?